corehq/apps/accounting/decorators.py

```diff
@@ -111,7 +111,7 @@ def wrapped(request, *args, **kwargs):
                 error_message = "You have lost access to this feature."
                 response = get_response(error_message, http_status_code)
                 return HttpResponse(json.dumps(response),
-                                    content_type="application/json")
+                                    content_type="application/json", status=401)
```

Comment:
if its not added as a kwarg to the HttpResponse the response returns with status code 200, which felt wrong to me. I included it in the json response, because that seems to be our convention elsewhere, at least with the decorator that returns json on unauthorized hits. This creates consistency between tastypie and non tastypie APIs we have. As for why its there in the first place, i assumed it was so you could pull the response code programmatically from the JSON if it returns an unexpected format, instead of having to rerun your request and search through the headers